src/components/InRequest/InRequestEditPanel.tsx

```diff
@@ -102,61 +103,19 @@ export const InRequestEditPanel: FunctionComponent<IInRequestEditPanel> = (
     reset(transRes);
   };
 
-  const updateRequest: SubmitHandler<IInRequest> = (data) => {
-    const requestApi = RequestApiConfig.getApi();
-    /* Validation has passed, so update the request */
-
-    // Transform "yes" / "no" to true/false
-    const hasExistingCAC = data?.hasExistingCAC ? true : false;
-    const isNewCivMil = data?.isNewCivMil ? true : false;
-    const isNewToBaseAndCenter = data?.isNewToBaseAndCenter ? true : false;
-    const isTraveler = data?.isTraveler ? true : false;
```

Comment:
Looking at this logic now, it should have had === "yes" on the left hand side to properly translate yes/no to true/false